### ISSUE TITLE
Captured Mouse Events

### ISSUE THREAD

**Issue by eladalon1983** [WebKit/standards-positions#233]
### WebKittens

@youennf

### Title of the spec

Captured Mouse Events

### URL to the spec

https://screen-share.github.io/captured-mouse-events/

### URL to the spec's repository

https://github.com/screen-share/captured-mouse-events/

### Explainer URL

https://github.com/screen-share/captured-mouse-events/blob/main/README.md

### Description

Web applications can use getDisplayMedia() to capture any display-surface - tabs, windows or screens. When they do, they can also specify the [cursor constraint](https://www.w3.org/TR/screen-capture/#dfn-cursor) to control whether the cursor's pixels are captured or not.

But what if the application wishes to programmatically observe the location of the cursor? That can be done by scanning each frame and employing heuristics to detect the cursor. But that's neither simple, nor efficient, nor robust.

To that end, we propose a mechanism for exposing mouse coordinates over a captured surface to a capturing application.

The TL;DR is that we expose an oncapturedmousechange EventHandler on [CaptureController](https://www.w3.org/TR/screen-capture/#dom-capturecontroller). Events are of the general shape {surfaceX: long, surfaceY: long}, exposing the coordinates of the mouse relative to the origin of the captured surface.
